packages/valory/skills/transaction_settlement_abci/behaviours.py

```diff
@@ -165,6 +180,165 @@ def get_gas_price_params(self, tx_body: dict) -> List[str]:
 
         return []
 
+    def _send_transaction_request(
+        self,
+        signing_msg: SigningMessage,
+        use_flashbots: bool = False,
+        target_block_numbers: Optional[List[int]] = None,
```

Comment:
As far as I can tell these two args are never passed.

Reply:
@dvilelaf Introduced this, https://github.com/valory-xyz/iekit/blob/f345fb8806413fe145fdab3e479eecf0401306d7/packages/valory/skills/transaction_settlement_abci/behaviours.py#L306-L308 he is also using these arguments